index.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Movies</title>
+    <link rel="stylesheet" href="./style.css" />
+
+    <!--Google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Fredoka&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Google Icons -->
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0"
+    />
+
+    <script defer src="index.js"></script>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <h3>IMOX</h3>
+        <ul class="nav-list">
+          <li class="nav-item active">Home</li>
+          <li class="nav-item">Movies</li>
+          <li class="nav-item">Contact</li>
+        </ul>
+        <div class="nav-extra-items">
+          <input type="text" placeholder="Search" />
+          <button>SIGN UP</button>
+        </div>
+        <div class="hamburger">
+          <span class="material-symbols-outlined"> menu </span>
+        </div>
+      </nav>
+
+      <div class="hero-content">
+        <h1 class="hero-heading">
+          RAYA AND THE <br />
+          LAST DRAGON
+        </h1>
+        <p class="hero-title">
+          Lorem ipsum, dolor sit amet consectetur adipisicing elit. Earum,
+          dolorum.
+        </p>
+        <p class="hero-desc">
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Laborum
+          facilis expedita id voluptatum!
+        </p>
+        <a href="./booking-page/index.html" class="ticket-btn">BUY TICKETS</a>
+      </div>
+    </header>
+
+    <main>
+      <section class="most-viewed">
+        <h2>Most Viewed</h2>
+        <div class="movies-container">
+          <div class="view-all">
+            <a href="" class="view-all-btn">View all</a>
+            <a href="" class="view-all-btn"
+              ><span class="material-symbols-outlined"> arrow_right_alt </span>
+            </a>
+          </div>
+          <div class="movies">
+            <div class="movie">
+              <img src="./img/hp.jpg" width="200" alt="" />
+              <h3>Harry Potter</h3>
+            </div>
+            <div class="movie">
```

Comment:
Use array or any object to print these divs. make them dynamic.

Reply:
mayank is working on it. he is handling API part.